dcos/package.py

```diff
@@ -1384,6 +1384,31 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_revision(self, package_version=None):
```

Comment:
Yea, there was some discussion on naming, if you scroll to the original comments on the pr. I think we settled on `package-version` to mean `software-version` and `package-revision` for `package number` (ie what is currently `package-version`.